drivers/video/video_stm32_dcmi.c

```diff
@@ -164,6 +269,8 @@ static int stm32_dma_init(const struct device *dev)
 						config->dma.channel);
 #elif defined(CONFIG_SOC_SERIES_STM32L4X)
 	hdma.Instance = __LL_DMA_GET_CHANNEL_INSTANCE(config->dma.reg, config->dma.channel);
+#elif defined(CONFIG_SOC_SERIES_STM32U5X)
+	hdma.Instance = LL_DMA_GET_CHANNEL_INSTANCE(config->dma.reg, config->dma.channel);
```

Comment:
This shouldn't be required after rebase on top of https://github.com/zephyrproject-rtos/zephyr/pull/94869